README.md

```diff
@@ -98,14 +98,31 @@ class MyFunction implements RequestHandler<String, String> {
     }
 ```
 
-### Support Java 11 and Above 
+### Support Java 11 and Above
 
 Add the environment variable `JAVA_TOOL_OPTIONS` to your Lambda functions and set it to
 `-Djdk.attach.allowAttachSelf=true` in addition to the manual code mentioned above.
 
 ### Supported Instrumentation Libraries
 
-- Aws SDK V1 
+- Aws SDK V1
 - Aws SDK V2
 - Apache HTTP Client
-- Apache Kafka
\ No newline at end of file
+- Apache Kafka
+
+### Secret scrubbing
+
+The tracer will automatically scrub values for keys matching the following regex patterns:
+- `.*pass.*`
+- `.*key.*`
+- `.*secret.*`
+- `.*credential.*`
+- `.*passphrase.*`
+- `SessionToken`
+- `x-amz-security-token`
+- `Signature`
+- `Authorization`
+
+from the payload, at any depth.
```

Comment:
I think we should give an example of a `LUMIGO_SECRET_MASKING_REGEX` value, original payload and masked payload.

I show in the example the
* root + sub fields are masked
* fields that only match a sub section of the regex are not masked
* The regular expressions are case sensitive (headers are case insensitive, so the `Authorization` header might be `authorization`)